scipy/interpolate/_rbfinterp.py

```diff
@@ -387,6 +389,73 @@ def __init__(self, y, d,
         self.epsilon = epsilon
         self.powers = powers
 
+    def _chunk_evaluator(
+            self,
+            x,
+            y,
+            shift,
+            scale,
+            coeffs,
+            memory_budget=None
+    ):
+        """
+        Evaluate the interpolation why controlling memory consumption.
+        We chunk the input if we need more memory than specified.
+
+        Parameters
+        ----------
+        x : (Q, N) float ndarray
+            array of points on which to evaluate
+        y: (P, N) float ndarray
+            array of points on which we know function values
+        shift: (N, ) array
```

Comment:
nit: sometimes it's `array`, sometimes it's `ndarray`. Did you mean `array_like` (i.e., can be a list and you wrap it in an `np.asarray`)? OTOH, it's an internal function, so this is not high-prio

Reply:
I've changed the comment to ndarray (without trying the np.asarray conversions)